sqllogictest/src/runner.rs

```diff
@@ -1023,11 +1023,9 @@ impl<D: AsyncDB, M: MakeConnection<Conn = D>> Runner<D, M> {
             // for every slt file, we create a database against table conflict
             let file = file.unwrap();
             let db_name = file
-                .file_name()
-                .expect("not a valid filename")
                 .to_str()
-                .expect("not a UTF-8 filename");
-            let db_name = db_name.replace([' ', '.', '-'], "_");
+                .ok_or_else(|| anyhow!("not a UTF-8 filename"))?;
```

Comment:
Should not anyhow here. Returns `ParallelTestError`

Reply:
I'm comfortable with panicking here. Since non-UTF8 file name is quite rare.

Reply:
OK.